packages/core/config/storybook/preview.example.js

```diff
@@ -0,0 +1,8 @@
+// This is an example of a custom preview.js file. If you'd like to
```

Comment:
I created this file since `~__REDWOOD__USER_STORYBOOK_PREVIEW_CONFIG` should always be aliased to a valid filepath, otherwise HMR complains when we `require` it. This is essentially a no-op file to alias it to if the user doesn't have their own config file.